### PR TITLE
#5 comment count added to article object

### PR DESCRIPTION
**Pull request by BabakSebo**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+        expect(body.article).toEqual(
+          expect.objectContaining({
+            article_id: 1,
+            title: "Living in the shadow of a great man",
+            topic: "mitch",
+            author: "butter_bridge",
+            body: "I find this existence challenging",
+            created_at: expect.any(String),
+            votes: 100,
+            comment_count: expect.any(Number),
```
Comment (daviidbartlett) [inline __tests__/app.test.js:168]: it's not any number, you should assert how many comments you expect this to have

Comment (daviidbartlett) [reply __tests__/app.test.js:168]: also you should need to test the whole object again just this property

File: model.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+exports.selectTopics = () => {
+  return db.query(`SELECT * FROM topics`).then(({ rows }) => {
+    return rows;
+  });
+};
+
+exports.selectArticlesById = (id) => {
+  return db
+    .query(
+      `SELECT articles.*, COUNT(comments.comment_id)::INT
```
Comment (daviidbartlett) [inline model.js:12]: no body thanks!

File: model.js [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+};
+
+exports.selectUsers = () => {
+  return db.query("SELECT * FROM users").then(({ rows }) => {
+    return rows;
+  });
+};
+
+exports.selectArticles = () => {
+  return db.query("SELECT * FROM articles").then(({ rows }) => {
```
Comment (daviidbartlett) [inline model.js:46]: no body thanks! also there should be an order to it

File: model.js [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+      "UPDATE articles SET votes = votes + $1 WHERE article_id = $2 RETURNING *",
+      [newVotes, id]
+    )
+    .then(({ rows }) => {
+      return rows[0];
+    });
+};
+
+exports.selectUsers = () => {
+  return db.query("SELECT * FROM users").then(({ rows }) => {
```
Comment (daviidbartlett) [inline model.js:40]: we don't want everything for users

**Review by daviidbartlett — COMMENTED** [review]
Why is all of this code new? There should only be a couple of lines and a new test for this PR. Make sure once they've been accept you pull them down locally to main and branch off from the new main

File: __tests__/app.test.js (hunk 1 above)
Comment (BabakSebo) [reply __tests__/app.test.js]: specific comment count added
object properties removed bar comment_count

File: model.js (hunk 3 above)
Comment (BabakSebo) [reply model.js]: model amended, body not included

File: model.js (hunk 4 above)
Comment (BabakSebo) [reply model.js]: model amended to only return username

**Pull request merged**